### DISCUSSION TITLE
is apt install tur-repo enough

### DISCUSSION THREAD

**Discussion by tathastu871** [termux-user-repository/tur#283] [Q&A]
how to setup hacking continous and other components

**Comment by T-Dynamos** [comment]
adding `tur-hacking` to `$PREFIX/etc/apt/sources.list.d/tur.list` would make you install hacking packages.

**Reply by tathastu871** [reply]
```printf "%s\n" "tur-multilib" "tur-continous" "tur-hacking" "tur-on-device" >> $PREFIX/etc/apt/sources.list.d/tur.list``` 
will it work.
do i nned to do above after apt install tur-repo
